src/tests/CMakeLists.txt

```diff
@@ -54,6 +54,7 @@ EkatCreateUnitTest(mam4_wet_deposition_unit_tests mam4_wet_deposition_unit_tests
 EkatCreateUnitTest(mam4_mo_setsox_unit_tests mam4_mo_setsox_unit_tests.cpp
   LIBS mam4xx_tests ${HAERO_LIBRARIES} EXCLUDE_TEST_SESSION)
 
+if (NOT ${CMAKE_CXX_COMPILER_ID} STREQUAL "Intel") #The Intel compiler always emit a warning for inline functions.
 target_compile_options(utils_unit_tests PRIVATE -Werror)
```

Comment:
One last thing I didn't notice till now. Can you indent the statements inside this `if` block using 2 spaces? Thanks!